Alex/12/AOC2021_12B_v00.py

```diff
@@ -67,88 +67,51 @@ def can_i_visit_this_small_cave(small_cave,path_count):
 			if cave.islower() is True and cave_occurances >= 2:
 				#Another cave has been visited twice. We cannot visit our small cave again.
 				visit_small_cave_bool = False
-		
-		if visit_small_cave_bool == '':
-			#no other small cave has occured twice. We may visit this one again.
-			visit_small_cave_bool = True
-		
-	else:
-		#Small cave has not been visited. You may visit it
-		visit_small_cave_bool = True
-	
-	
-	
 	return visit_small_cave_bool
 
 
-
-
-def produce_new_paths(path,nodesdict,paths,path_count):

```

Comment:
'paths' wasn't used - removed
'path_count' - moved into the function so that it's only calculated if needed.